spec/spec_helper.rb

```diff
@@ -159,3 +160,6 @@ def quote_column_name(name)
 CONNECTION_PARAMS_WITH_CATALOG = CONNECTION_PARAMS.merge(:catalog => CATALOG_FILE)
 
 ActiveRecord::Base.establish_connection(AR_CONNECTION_PARAMS)
+
+Rake.application.init
+load File.expand_path('../../Rakefile', __FILE__)
```

Comment:
If recreation of `sales` data will be done as suggested above then this can be removed.